src/host_factory.tokens.create.md

```diff
@@ -33,7 +33,7 @@ Generate multiple tokens at once with the `count` parameter. By default, one tok
 |404|Host factory not found|
 
 + Parameters
-    + id: redis%2Fdefault (string) - ID of the host factory, query-escaped
+    + id: redis/default (string) - ID of the host factory, query-escaped
```

Comment:
I'm confused. Wasn't that the whole change?

Reply:
i think the change is marking expiration as not optional and CIDR as optional.

ID needs to be `redis/default` or else the HF API tests don't pass.